### ISSUE TITLE
Loading saves in example games causes a stack trace

### ISSUE THREAD

**Issue by itsbirdseed** [irmen/Tale#33]
Loading a previously saved game gives you an error and dumps you in limbo. It looks like the problem cropped up with 0a78d4a821d7da405bf5335ba3741757f0b593fe.

[Here's the stack trace from Zed is Me.](https://github.com/irmen/Tale/files/2613505/error.txt)

**Comment by irmen** [comment]
When you say "previously saved game" is that a save game from an older version of Tale, or a savegame you created with the exact same tale version?

**Comment by itsbirdseed** [comment]
Yes, both latest. It should be reproducible by opening a game, immediately saving and quitting, and opening and loading again. I originally tried this on my linux machine, so I installed the newest version of python on windows just to be sure and got the same result. Let me know if there's anything else you need to know.

**Comment by irmen** [comment]
Thanks for the clarification. I'll look into it.
